lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -602,6 +602,8 @@ export abstract class ClientApplication {
         commonRequest: CommonSilentFlowRequest,
         silentRequest: SilentRequest
     ): Promise<AuthenticationResult> {
+        const preQueueTime = this.browserStorage.getPreQueueTime(PerformanceEvents.AcquireTokenFromCache, commonRequest.correlationId);
```

Comment:
Would it be possible to combine `getPreQueueTime` and `addQueueMeasurement` into a single method? These methods are always being invoked one after another.

Reply:
Thanks for this feedback. Updated the pattern and combined these two into one method.